verification/api/handler.go

```diff
@@ -472,3 +475,84 @@ func sendChallengeResponseSessionCreated(c *gin.Context, id string, jsonSession
 	c.Header("Location", path.Join("session", id))
 	sendChallengeResponseSessionWithStatus(c, http.StatusCreated, jsonSession)
 }
+
+func (o *Handler) getKey() (jwk.Key, error) {
+	var key jwk.Key
+
+	protoKey, err := o.Verifier.GetPublicKeys()
```

Comment:
MIA from my previous review:

```suggestion
	protoKey, err := o.Verifier.GetPublicKey()
```